### PR TITLE
Add TextAlign to tooltip.

### PR DESCRIPTION
**Pull request by terryl1900**
As title says, which is a feature I need. Added to both Line and Bar chart.

### CODE REVIEW

File: repo_files/documentations/bar_chart.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ enum values {`start`, `end`, `center`, `spaceEvenly`, `spaceAround`, `spaceBetwe
 |:-------|:----------|:------------|
 |text|text string of each row in the tooltip bubble|null|
 |textStyle|[TextStyle](https://api.flutter.dev/flutter/dart-ui/TextStyle-class.html) of the showing text row|null|
+|textAlign|[TextAlign](https://api.flutter.dev/flutter/dart-ui/TextAlign-class.html) of the showing text row|null|
```
Comment (imaNNeoFighT) [inline repo_files/documentations/bar_chart.md:102]: Please put `TextAlign.center` on the default value section. it is not null.

File: repo_files/documentations/line_chart.md [hunk 2]
```diff
@@ -186,6 +186,7 @@ LineChart(
 |:-------|:----------|:------------|
 |text|text string of each row in the tooltip bubble|null|
 |textStyle|[TextStyle](https://api.flutter.dev/flutter/dart-ui/TextStyle-class.html) of the showing text row|null|
+|textAlign|[TextStyle](https://api.flutter.dev/flutter/dart-ui/TextAlign-class.html) of the showing text row|null|
```
Comment (imaNNeoFighT) [inline repo_files/documentations/line_chart.md:189]: Please put `TextAlign.center` on the default value section. it is not null.

File: repo_files/documentations/bar_chart.md (hunk 1 above)
Comment (terryl1900) [reply repo_files/documentations/bar_chart.md]: Done. Good catch.

File: repo_files/documentations/line_chart.md (hunk 2 above)
Comment (terryl1900) [reply repo_files/documentations/line_chart.md]: Done. Good catch.

**Comment by imaNNeoFighT** [comment]
Thanks for your contribution :)
Hope to see more from you.
Good job!

**Pull request merged**